gui/src/renderer/components/SearchBar.tsx

```diff
@@ -71,10 +71,12 @@ interface ISearchBarProps {
   onSearch: (searchTerm: string) => void;
   className?: string;
   disableAutoFocus?: boolean;
+  searchInputRef?: React.Ref<HTMLInputElement>;
 }
 
 export default function SearchBar(props: ISearchBarProps) {
```

Comment:
In your [previous suggestion](https://github.com/mullvad/mullvadvpn-app/pull/6832#issuecomment-2371063363) @raksooo , you showed me `AutoSizingTextInputWithRef` as an example:

https://github.com/mullvad/mullvadvpn-app/blob/d35866d337647a26dc8ffc2e875212306dd7c0ce/gui/src/renderer/components/cell/Input.tsx#L196

This components consumes a `forwardedRef` parameter, the `ref` object does not come from its `props`.
But here I was only able to pass it via a prop.